tests/integration/api_v3/constants/invalidInputs.js

```diff
@@ -0,0 +1,107 @@
+/*
+ * LiskHQ/lisk-service
+ * Copyright © 2023 Lisk Foundation
+ *
+ * See the LICENSE file at the top-level directory of this distribution
+ * for licensing information.
+ *
+ * Unless otherwise agreed in a custom licensing agreement with the Lisk Foundation,
+ * no part of this software, including this file, may be copied, modified,
+ * propagated, or distributed except according to the terms contained in the
+ * LICENSE file.
+ *
+ * Removal or modification of this copyright notice is prohibited.
+ *
+ */
+
+const invalidAddresses = [
+	'lsk12345678901234567890123456789012345678901', // length exceeds 40 characters
+	'Lsk1234567890123456789012345678901234567890', // starts with 'L' instead of 'lsk'
```

Comment:
Did you intend it to be like this instead?
```suggestion
	'L1234567890123456789012345678901234567890', // starts with 'L' instead of 'lsk'
```